trace/cloud-trace-demo-app/app/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8.0
+
+RUN pip3 install --upgrade pip
+
+RUN apt-get update \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /app/
+
+WORKDIR /app
+RUN pip3 install -r requirements.txt
+RUN pip3 install opencensus-ext-stackdriver
```

Comment:
Why are these requirements listed outside `requirements.txt`?

I think it's also better to chain dependencies so it's a single build step:
```
# Install production dependencies.
RUN set -ex; \
    pip install -r requirements.txt; \
    pip install opencensus-ext-stackdriver \
    pip install grpcio
```

Reply:
I moved those dependencies into requirements.txt